### ISSUE TITLE
Suave.Experimental chokes in F# Interactive

### ISSUE THREAD

**Issue by leolorenzoluis** [SuaveIO/suave#518]
Given the following code where the products has 500 items.

```
let details (products : seq<Product>) = [
        for product in products ->
            div [
                ]
    
]
```

I get the following error

Session termination detected. Press Enter to restart.

If I reduce the number of items to say 50-100, it works fine. Is this a limitation in F# Interactive or it's just the space of using Suave.Html is very limited?

**Comment by ademar** [comment]
Seems like a F# interactive limitation.

**Comment by dsyme** [comment]
@leolorenzoluis If you'd like to put an issue at http://github.com/Microsoft/visualfsharp it's possibe we could do something about it

**Comment by leolorenzoluis** [comment]
@dsyme I don't know what happened but when I tried to recreate it while I'm writing the bug in VisualFSharp, I suddenly cannot reproduce it. (I guess that's a good thing for now...)

I don't know what happened, at first I thought it was some limitation when trying to print out large number of objects in one statement with F# Interactive, but it looks like it can print extremely large number of characters. 

Could it be that I recently updated my Mono through Xamarin Studio, but I'm not sure if that'll affect F# Interactive?

By the way, I'm using macOS.
